javalib/src/main/scala/java/util/zip/ZipEntry.scala

```diff
@@ -77,26 +77,30 @@ class ZipEntry private (
     size
 
   def getTime(): Long = {
-    if ((time == -1) || (modDate == -1)) -1L
-    else
-      synchronized {
-        val tm = stackalloc[tm]()
+    // Revert PR #3794 so I can chase intermittent bad values & Segfault
```

Comment:
I think that just ingoring the 2 problematic tests is good enough. We can temporary keep the current implementation, because it works, but only has some issues in specific environments

Reply:
Your call.  

I never quite trust intermittent bugs which could be memory spray.  I'll rest better if I know that
I have not introduced a bug which could pop up an hour before the 0.5.0 release or on my next PR.

The previous behavior is long standing. The driver/motivator for the PR which introduced the
change is `jdk.zipfs` work, not any direct user Issue with someone waiting for a fix.

I am trying to reproduce the issue locally.